packages/discord.js/src/structures/Embed.js

```diff
@@ -2,11 +2,28 @@
 
 const { Embed: BuildersEmbed } = require('@discordjs/builders');
 const Transformers = require('../util/Transformers');
+const { Util } = require('../util/Util');
 
+/**
+ * Represents an embed object
+ */
 class Embed extends BuildersEmbed {
   constructor(data) {
     super(Transformers.toSnakeCase(data));
   }
+
+  /**
+   * Sets the color of this embed
+   *
+   * @param {ColorResolvable} color The color of the embed
```

Comment:
```suggestion
   * @param {ColorResolvable} color The new color of the embed
```

Reply:
Sorry, but the docs are currently mirroring the way it's documented in the builders package. Going to keep this the same.

Reply:
There’s always room for improvement and having it like this makes it seem like it’s the current color but alright 🤔